dbt/adapters/snowflake/impl.py

```diff
@@ -83,7 +83,7 @@ def _behavior_flags(self) -> List[BehaviorFlag]:
         return [
             {
                 "name": "enable_iceberg_materializations",
-                "default": False,
+                "default": True,
```

Comment:
Will this be turned back off before merge?

Reply:
Yeah, that's how I was testing that things worked instead of duplicating all the tests. Now that I streamlined the tests, I will update them to use user overrides instead. Before I would have had to duplicate ~20 tests.